core.go

```diff
@@ -96,35 +96,46 @@ func Strict(laxValidator validator.Validator) validator.Validator {
 	}
 }
 
+var strType = reflect.TypeOf("")
```

Comment:
Are these unused?

Reply:
yes, apologies, I missed that, that was left over from an experiment.